auth_config.py

```diff
@@ -0,0 +1,11 @@
+from pydantic import Field, BaseModel
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+class AuthProfile(BaseModel):
+    username: str
+    password: str
+
+class AuthConfig(BaseSettings):
+    profiles: dict[str, AuthProfile] = Field(alias="auth")
```

Comment:
That's a good idea! However, I am a bit hesitant to implement env variable parsing like this because the plaintext password might also start with a `$` sign. Will come up with a solution for that.

Reply:
It's now implemented in dfdddca5ae2576d779d6b38696487d6d2118791d

Reply:
looks much better :)